client/src/ui/BitbakeRecipesView.ts

```diff
@@ -135,4 +146,8 @@ class BitbakeTreeDataProvider implements vscode.TreeDataProvider<BitbakeRecipeTr
     item.tooltip = 'Add a recipe to the active workspace'
     return item
   }
+
+  private getRecipeInfo (label: string): ElementInfo | undefined {
```

Comment:
This is replaced by the `recipeInfoMap`, do we still need to keep it?

Reply:
No longer used! good catch. Removing it.